assets/css/style.css

```diff
@@ -172,31 +300,39 @@ input[type="checkbox"] {
 
 .calendar {
   background-color: #9fbe6b;
+  border: none;
+  border-radius: .4rem 0 0 .4rem;
   width: 3rem;
   height: 3rem;
-  border: none;
   cursor: pointer;
   position: absolute;
   top: 2.4rem;
   left: -3rem;
+  transition-duration: .4s;
+}
+
+.calendar:hover {
+  width: 10rem;
+  left: -10rem;
+  transition-duration: .4s;
```

Comment:
The hover effect is very cool!
Is the transition-duration property necessary here? I think including it in the ruleset for .calendar is sufficient? Please correct me if I'm wrong.

Reply:
It's whenever the new one is active. So you need one in both directions. If you remove it and hover it will pop open instantly and if you remove the one in .calendar it will shrink instantly.

Reply:
ah I got it. Thank you! I had to increase the value of transition-duration to 2s to see the differences lol

Reply:
:)